nodejs/20/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM        --platform=$TARGETOS/$TARGETARCH node:20-bookworm-slim
+FROM        --platform=$TARGETOS/$TARGETARCH node:20-slim
```

Comment:
keep this to bookworm, unless you have a real valid reson to not.

We lock it for a reson as we had it brake in the past with os updates (11->12)